mlflow/utils/model_utils.py

```diff
@@ -0,0 +1,32 @@
+import os
+
+from mlflow.exceptions import MlflowException
+from mlflow.models import Model
+from mlflow.protos.databricks_pb2 import RESOURCE_DOES_NOT_EXIST
+
+def _get_flavor_configuration(model_path, flavor_name):
```

Comment:
Can you add unit test coverage for this function? Error and passing conditions

Reply:
Added a `tests/utils/test_model_utils.py` suite with tests for the error and passing conditions associated with this method.